Makefile

```diff
@@ -221,7 +221,7 @@ test: test-unit lint
 # The presubmits have made side-effects in the past, which makes their
 # validation suspect (as the repository they are validating is different
 # than what is checked in).  Run `test` but verify that repository is clean.
-test-presubmit: test
+test-presubmit: test tidy
```

Comment:
Can you add an `all` target with the tidy so we get it automatically without having to run `make tidy` or `make test-presubmit`? Then remove the default so it reverts to make all being default instead of make test.

Reply:
I'm not saying that actually should be the behavior, just that it's what I might expect if I was running a `make all` target without context. I don't feel that strongly about it so I can make the change you suggest.

Take a look at https://github.com/GoogleContainerTools/kpt-config-sync/pull/148 for some more changes aimed at improving the developer experience with the make targets

Reply:
Done